### PR TITLE
[docs] Added docs for gossip encryption config

### PR DESCRIPTION
**Pull request by tradel**
Release 0.7.0 of the Helm chart includes support for gossip encryption. This PR documents the new configuration options.

### CODE REVIEW

File: website/source/docs/platform/k8s/helm.html.md [hunk 1]
```diff
@@ -84,6 +84,14 @@ and consider if they're appropriate for your deployment.
 
   * <a name="v-global-datacenter" href="#v-global-datacenter">`datacenter`</a> (`string: "dc1"`) - The name of the datacenter that the agent cluster should register as. This may not be changed once the cluster is bootstrapped and running, since Consul doesn't yet support an automatic way to change this value.
 
+  * <a name="v-global-gossip" href="#v-global-gossip">`gossipEncryption`</a> - Configuration to enable [Gossip encryption](https://www.consul.io/docs/agent/encryption.html#gossip-encryption). Use of gossip encryption is highly recommended in production environments.
```
Comment (kaitlincarter-hc) [inline website/source/docs/platform/k8s/helm.html.md]:
```suggestion
  * <a name="v-global-gossip" href="#v-global-gossip">`gossipEncryption`</a> - Provides the ability to enable [Gossip encryption](https://www.consul.io/docs/agent/encryption.html#gossip-encryption). Use of gossip encryption is highly recommended in production environments.
```

File: website/source/docs/platform/k8s/helm.html.md [hunk 2]
```diff
@@ -84,6 +84,14 @@ and consider if they're appropriate for your deployment.
...
+
+      - <a name="v-global-gossip-enabled" href="#v-global-gossip-enabled">`enabled`</a> (`boolean: true`) - If true, gossip traffic between agents will be encrypted.
+
+      - <a name="v-global-gossip-secretname" href="#v-global-gossip-secretname">`secretName`</a> (`string: null`) - The name of the Kubernetes secret that contains the Gossip gncryption key. This defaults to null.
```
[the remark this answers is not in the document]
Comment (tradel) [reply website/source/docs/platform/k8s/helm.html.md]: Fixed by 4e4fa3209219637ff88c22c65d8d9228546257e2.

File: website/source/docs/platform/k8s/helm.html.md [hunk 3]
```diff
@@ -84,6 +84,14 @@ and consider if they're appropriate for your deployment.
 
   * <a name="v-global-datacenter" href="#v-global-datacenter">`datacenter`</a> (`string: "dc1"`) - The name of the datacenter that the agent cluster should register as. This may not be changed once the cluster is bootstrapped and running, since Consul doesn't yet support an automatic way to change this value.
 
+  * <a name="v-global-gossip" href="#v-global-gossip">`gossipEncryption`</a> - Configuration to enable [Gossip encryption](https://www.consul.io/docs/agent/encryption.html#gossip-encryption). Use of gossip encryption is highly recommended in production environments.
+
+      - <a name="v-global-gossip-enabled" href="#v-global-gossip-enabled">`enabled`</a> (`boolean: true`) - If true, gossip traffic between agents will be encrypted.
```
[the remark this answers is not in the document]
Comment (tradel) [reply website/source/docs/platform/k8s/helm.html.md]: The `enabled` option has been removed entirely.

File: website/source/docs/platform/k8s/helm.html.md (hunk 1 above)
Comment (tradel) [reply website/source/docs/platform/k8s/helm.html.md]: Fixed by 4e4fa3209219637ff88c22c65d8d9228546257e2.

File: website/source/docs/platform/k8s/helm.html.md [hunk 4]
```diff
@@ -84,6 +84,12 @@ and consider if they're appropriate for your deployment.
 
   * <a name="v-global-datacenter" href="#v-global-datacenter">`datacenter`</a> (`string: "dc1"`) - The name of the datacenter that the agent cluster should register as. This may not be changed once the cluster is bootstrapped and running, since Consul doesn't yet support an automatic way to change this value.
 
+  * <a name="v-global-gossip" href="#v-global-gossip">`gossipEncryption`</a> - Provides the ability to enable [Gossip encryption](https://www.consul.io/docs/agent/encryption.html#gossip-encryption). Use of gossip encryption is highly recommended in production environments. You must specify both `secretName` and `secretKey` to enable gossip encryption.
```
Comment (judithpatudith) [inline website/source/docs/platform/k8s/helm.html.md]:
```suggestion
  * <a name="v-global-gossip" href="#v-global-gossip">`gossipEncryption`</a> - Provides the ability to enable [Gossip encryption](https://www.consul.io/docs/agent/encryption.html#gossip-encryption). We highly reccomend that you enable gossip encryption in all production environments. You must specify both `secretName` and `secretKey` to enable gossip encryption.
```

File: website/source/docs/platform/k8s/helm.html.md [hunk 5]
```diff
@@ -84,6 +84,12 @@ and consider if they're appropriate for your deployment.
 
   * <a name="v-global-datacenter" href="#v-global-datacenter">`datacenter`</a> (`string: "dc1"`) - The name of the datacenter that the agent cluster should register as. This may not be changed once the cluster is bootstrapped and running, since Consul doesn't yet support an automatic way to change this value.
 
+  * <a name="v-global-gossip" href="#v-global-gossip">`gossipEncryption`</a> - Provides the ability to enable [Gossip encryption](https://www.consul.io/docs/agent/encryption.html#gossip-encryption). We highly reccomend that you enable gossip encryption in all production environments. You must specify both `secretName` and `secretKey` to enable gossip encryption.
```
Comment (kaitlincarter-hc) [inline website/source/docs/platform/k8s/helm.html.md]:
```suggestion
  * <a name="v-global-gossip" href="#v-global-gossip">`gossipEncryption`</a> - Provides the ability to enable [Gossip encryption](/docs/agent/encryption.html#gossip-encryption). We highly recommend that you enable gossip encryption in all production environments. You must specify both `secretName` and `secretKey` to enable gossip encryption.
```

**Comment by i0rek** [comment]
This is already in the docs.

**Pull request closed without merging**